packages/storybook-vue/stories/3_components/table/Table.stories.mdx

```diff
@@ -380,3 +382,77 @@ Please switch to the Canvas view for a working example
   }
 </script>
 ```
+
+## With striped rows
+
+<Canvas withSource="none">
+  <Story
+    name="With striped rows"
```

Comment:
Let's have the title capitalised, for consistency: "With Striped Rows" 🙏

Reply:
Thanks for the hint! adapted